extensions/amp-bind/0.1/amp-state.js

```diff
@@ -48,8 +49,11 @@ export class AmpState extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
-    user().assert(isExperimentOn(this.win, 'amp-bind'),
-        `Experiment "amp-bind" is disabled.`);
+    // Don't check experiment when testing.
```

Comment:
Sorry I probably should have clarified. I think a comment explaining why this is happening would be beneficial here.